packages/react-examples/src/react-label/Label/Label.stories.tsx

```diff
@@ -55,60 +56,67 @@ export const CustomizableLabelExample = () => {
   const [required, setRequired] = React.useState(false);
   const [size, setSize] = React.useState<LabelProps['size']>('medium');
 
-  const updateLabelText = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, val?: string) => {
-    if (val) {
-      setLabelText(val);
-    }
+  const updateLabelText = (ev: React.FormEvent<HTMLInputElement>) => {
+    setLabelText(ev.currentTarget.value);
   };
 
-  const updateRequiredText = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, val?: string) => {
-    setRequiredText(val);
+  const updateRequiredText = (ev: React.FormEvent<HTMLInputElement>) => {
+    setRequiredText(ev.currentTarget.value);
   };
 
-  const updateStrong = (ev?: React.FormEvent<HTMLElement | HTMLInputElement>, checked?: boolean) => {
-    setStrong(checked!);
+  const updateStrong = (ev: React.FormEvent<HTMLInputElement>) => {
+    setStrong(!strong);
   };
```

Comment:
(nit) you're not using the `ev` argument so you can delete it. Also, this function is fairly simple, so it could possibly be just inlined where it is used, if you agree. Same for the other update functions.

`onChange={() => setStrong(!strong)}`